### ISSUE TITLE
FURB152 should look at additional digits to reduce false positives

### ISSUE THREAD

**Issue by hauntsaninja** [astral-sh/ruff#9281]
For example, it'll trigger on the following program
```
x = 2.7139914879495697
```
I think the rule should avoid erroring if there are additional decimal digits that do not match, so:
```
bad1 = 2.7182
bad2 = 2.7183
good1 = 2.71824
good2 = 2.71820001
```

**Comment by charliermarsh** [comment]
Trying to figure out what the best approach is here. I may need to look at what other tools do. E.g., what if the user rounds at some decimal point?

**Comment by hauntsaninja** [comment]
Thanks! Issue description mentions how I think rounding should be done; I opened a PR for this at https://github.com/astral-sh/ruff/pull/9290
